vmdk_plugin/utils/test_util/VolumeCreationTestUtil.go

```diff
@@ -49,3 +50,121 @@ func DeleteVolume(name string, ip string) ([]byte, error) {
 func InvokeCommand(remoteHostIP string, cmd string) ([]byte, error) {
 	return exec.Command("/usr/bin/ssh", append(SSH_OPTS, "root@"+remoteHostIP, cmd)...).CombinedOutput()
 }
+
+// returns disk format of volume using Admin cli
+func GetDiskFormat_AdminCli(volName string, hostname string) string {
+	cmd := "/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py ls -c volume,disk-format | grep " + volName
+	io := ExecuteCmd(cmd, hostname)
+	if strings.Contains(io, "stty: standard input: Inappropriate ioctl for device") {
+		io = strings.Replace(io, "stty: standard input: Inappropriate ioctl for device", "", -1)
+	}
+	result := strings.Fields(io)
+	diskFormat := result[1]
+	return string(diskFormat)
+}
+
+// returns disk format of volume using docker cli
+func GetDiskFormat_DockerCli(volName string, hostname string) string {
+	cmd := "docker volume inspect " + volName
+	io := ExecuteCmd(cmd, hostname)
+	volProps := strings.Fields(io)
+	i := 0
+	for range volProps {
+		if volProps[i] == "\"diskformat\":" {
+			i = i + 1
+			break
+		}
+		i++
+	}
+	fmt.Println("disk format is  - ", volProps[i])
+	df := strings.Trim(volProps[i], "\"")
+	dskFormat := strings.Trim(df, ",")
+	result := strings.Trim(dskFormat, "\"")
+	fmt.Println("RESULT: ", result)
+	return result
+}
+
+// returns attached to vm field of volume using docker cli
+func GetVmAttachedToVol_DockerCli(volName string, hostname string) string {
+	cmd := "docker volume inspect " + volName
+	io := ExecuteCmd(cmd, hostname)
```

Comment:
this code has been copied atleast 3 times .. it should be properly organized by moving into separate util.